data/migrations/20200131203724_car-info.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('cars', tbl => {
+        tbl.increments();
```

Comment:
instead of using tbl, why not use the semantic value to represent the argument being passed to the function, which would be a "car"